conan/tools/google/bazeldeps.py

```diff
@@ -557,14 +600,27 @@ def __init__(self, conanfile):
 
     def generate(self):
         """
-        Generates all the targets <DEP>/BUILD.bazel files and the dependencies.bzl one in the
-        build folder. It's important to highlight that the dependencies.bzl file should be loaded
-        by your WORKSPACE Bazel file:
+        Generates all the targets <DEP>/BUILD.bazel files, a dependencies.bzl (for bazel<7), a
+        conan_deps_repo_rules.bzl and a conan_deps_module_extension.bzl file (for bazel>=7.1) one in the
+        build folder.
+
+        In case of bazel < 7, it's important to highlight that the dependencies.bzl file should
+        be loaded by your WORKSPACE Bazel file:
 
         .. code-block:: python
 
             load("@//[BUILD_FOLDER]:dependencies.bzl", "load_conan_dependencies")
             load_conan_dependencies()
+
+        In case of bazel >= 7.1, the conan_deps_module_extension.bzl file should be loaded by your
+        Module.bazel file, e.g. like this:
+
+        .. code-block:: python
+                load_conan_dependencies = use_extension(
+                    "//build:conan_deps_module_extension.bzl",
+                    "load_dependencies"
+                )
+                use_repo(load_conan_dependencies, "dep-1", "dep-2", ...)
```

Comment:
Then, the UX is something like this:

```bazel
load_conan_dependencies = use_extension("//conan:conan_deps_module_extension.bzl", "load_dependencies")
use_repo(load_conan_dependencies, "dep-1", "dep-2", ...)
```

So, is it not possible to make something generic as it was done before?

```bazel
load_conan_dependencies()
```

Reply:
Correct, with this approach this will not be possible.

But when I compare this to Cmake, there you also have duplicated information like this regularly. 
I would say it could be compared to a find_package call.

Reply:
Yeah, I agree with that 👍